src/components/DpSearchField/DpSearchField.stories.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta } from '@storybook/addon-docs'
+import DpSearchField from './DpSearchField'
+
+<Meta
+    title="Components/SearchField"
+    component={DpSearchField}
+/>
+
+# SearchField
+
+The DpSearchField component to add input field with attribute `type="search"` and a button to submit search processing.
+
+The DpSearchField component has two events `search & reset`.
+
+## Props:
+
+### inputWidth:
+
+Value has to be a css class like 'u-1-of-2'.
```

Comment:
As already discussed in another documentation PR: i'd suggest to not document props separately here. Instead, improve the prop docs in the component code itself, if needed.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/492/commits/969fb0c587ad25677a50fff83f41fc0f452b582e